extension/lib/BiasDoorhanger.jsm

```diff
@@ -146,7 +146,9 @@ class BiasDoorhanger {
 
       const stats = await DoorhangerStorage.getStats(hostname);
       const timeSinceShown = Date.now() - stats.timestamp;
-      const shouldShow = !stats.neverAgain && timeSinceShown > Config.showDoorhangerInterval;
+      const hiddenTooQuickly = timeSinceShown < 1000;
```

Comment:
Some websites make a quick change to the URL when a page loads adding things like "#something" etc. which causes the doorhanger to be immediately dismissed. This keeps that from happening.